### PR TITLE
enable magit-svn by default

### PR DESCRIPTION
**Pull request by hornuda**
Hello together,

with this PR magit-svn will be enabled by default.
I use this at work since weeks without issues.
Nevertheless thats's my first PR so I appreciate any comments about missed points.

Cheers,
David

PS:  Thank you guys for spacemacs - incredible piece.

### CODE REVIEW

**Comment by sdwolfz** [comment]
Hello @hornuda . I don't think non svn users will be verry happy about enabling this by default and I will like to not merge that change.

I see you did some other changes as well in the `git/init-magit-svn` function. Can you please detail the motivation behind those?

**Comment by hornuda** [comment]
Hello @sdwolfz, you are totally right, it will readd the git-enable-magit-svn-plugin parameter.

Sure, the motivation of changing `git/init-magit-svn` was that I could not get open the magit-svn-popup by pressing 'N'. Pressing 'N leads to the message 'Search failed: "No search pattern"'. Thats the reason I copied the  initialisation from gitflow and bound magit-svn-popup to '!'. Maybe i misunterstood the 'N' mapping ?

**Comment by sdwolfz** [comment]
If you press `SPC hdk N` in a magit buffer and it's bound to `evil-ex-search-previous` then it makes sense to change the keybinding. Also you can put it after `:config` and you won't need the `with-eval-after-load`.

File: layers/+source-control/git/README.org [hunk 1]
```diff
@@ -67,37 +63,18 @@ function, this is the folder where you keep all your git-controlled projects
 For more information, see [[http://magit.vc/manual/magit.html#Status-buffer][Magit-User-Manual#Status-buffer]]
 
 ** Magit SVN plugin
-For convenience the magit SVN plugin can be activated directly in the Git
-layer by setting the variable =git-enable-magit-svn-plugin= to =t=.
```
Comment (sdwolfz) [inline layers/+source-control/git/README.org]: You need to add these instructions back.

File: layers/+source-control/git/README.org [hunk 2]
```diff
@@ -67,37 +63,18 @@ function, this is the folder where you keep all your git-controlled projects
...
+
+The magit SVN plugin shows commits which were not pushed to svn yet in *Magit* buffer.
+Press ~!​~ *Magit* buffer to open the magit-svn-popup where you can dcommit to svn or rebase from svn.
```
Comment (sdwolfz) [inline layers/+source-control/git/README.org]: There are some spelling mistakes here.

File: layers/+source-control/git/README.org [hunk 3]
```diff
@@ -67,37 +63,18 @@ function, this is the folder where you keep all your git-controlled projects
...
-** Git-Flow
-Git-flow is a standardized branching pattern for git repositories with the aim
-of making things more manageable. While there are tools to assist with making
-this easier, these do nothing you couldn't do manually.
-
-Support requires installation of the git-flow extensions. Please reference their
-[[https://github.com/petervanderdoes/gitflow/wiki][installation page]] for assistance.
-
-** Org integration
-See the commentary section of the package [[https://github.com/magit/orgit/blob/master/orgit.el#L28][here]].
```
Comment (sdwolfz) [inline layers/+source-control/git/README.org]: This instructions are still relevant and need to be added back.

File: layers/+source-control/git/packages.el [hunk 4]
```diff
@@ -205,10 +204,9 @@ Press [_b_] again to blame further in the history, [_q_] to go up or quit."
     :if git-enable-magit-svn-plugin
     :commands turn-on-magit-svn
     :init (add-hook 'magit-mode-hook 'turn-on-magit-svn)
-    :config
-    (progn
-      (evil-define-key 'emacs magit-status-mode-map
-        "N" 'magit-key-mode-popup-svn))))
+    :config (progn
+              (spacemacs|diminish magit-svn-mode "SVN")
+              (define-key magit-mode-map "!" 'magit-svn-popup))))
```
Comment (sdwolfz) [inline layers/+source-control/git/packages.el]: Currently `!` is bound to `magit-run-popup`. Is there any other key you can think of to replace this?

**Comment by sdwolfz** [comment]
Thank you :heart:! 
And congratulations on your first Spacemacs PR :tada:!
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**